zowex/src/main.rs

```diff
@@ -80,15 +82,26 @@ struct DaemonResponse {
 fn main() -> std::io::Result<()> {
     // turn args into vector
     let mut _args: Vec<String> = env::args().collect();
+    let cmd_result: Result<i32, i32>;
 
     _args.drain(..1); // remove first (exe name)
 
-    let cmd_result: Result<i32, i32>;
+    // Do we only need to display our version?
+    if _args.len() >= 1 {
+        if _args[0] == "--version-exe" {
+            println!("{}", env!("CARGO_PKG_VERSION"));
+            std::process::exit(EXIT_CODE_SUCCESS);
+        }
+    }
```

Comment:
Is this the first instance where we run Daemon only options ?
If so, where are we thinking this should be documented ?
End-user: [docs.zowe.org](https://docs.zowe.org/) ?
Developers:  [`next` `/docs` folder](https://github.com/zowe/zowe-cli/tree/next/docs)

Reply:
I believe that it is the first case in which the EXE processes an option and never passes it to the daemon.

The main reason I implemented --version-exe was so that the "zowe daemon enable" command could call the EXE behind the scenes to get and display the version number of the EXE.

There is a file named zowe-cli\zowex\design.md, in which I plan to document the --version-exe option. We also have a section of doc named [Using Daemon Mode (Technical Preview)](https://docs.zowe.org/stable/user-guide/cli-using-daemon-mode/). That is where I think we would document it for end users during vNext pre-release. I think that vNext document will evolve into a section of the general CLI documentation in Zowe-Docs when we release V2.